script/encrypt-heroku-auth-token.js

```diff
@@ -1,4 +1,3 @@
-
 #!/usr/bin/env node
 
```

Comment:
The main change is deleting line 1, which is needed to fix linter errors. The other changes below are Prettier's reformatting.